README.md

```diff
@@ -143,6 +143,7 @@ library:
 | `build-type` | · | `Simple`, or `Custom` if `custom-setup` exists | Must be `Simple`, `Configure`, `Make`, or `Custom` | | |
 | `extra-source-files` | · | | Accepts [glob patterns](#file-globbing) | | |
 | `extra-doc-files` | · | | Accepts [glob patterns](#file-globbing) | | `0.21.2` |
+| `extra-files` | · | | Accepts [glob patterns](#file-globbing) | | `0.39.0` |
```

Comment:
As this is a very non-invasive change, do you think we should release as `0.38.1`?

NB: Adding a new field to `Hpack.Config.Package` I don't consider a "breaking" change:
- `Hpack.Config` is semi-internal
- A user should use `Hpack.Config.package` to construct a `Package` anyway, and if they did that, their code will continue to work just fine.
- In general, for records with a large number of fields, I as a user should always be prepared that new fields are added.

Reply:
Please also bump the package version accordingly, so that a release to Hackage is triggered